tests/test_timers.py

```diff
@@ -17,45 +17,41 @@ def tearDown(self):
             db.session.remove()
             db.drop_all()
 
-    def test_get_all_timers(self):
+    def test_get_one_timer(self):
```

Comment:
yay tests! These make sense. I'm glad the skills we learned using RSpec prove to be so helpful across different languages and frameworks.